paasta_tools/cli/schemas/adhoc_schema.json

```diff
@@ -180,6 +183,15 @@
                 },
                 "role": {
                     "type": "string"
+                },
+                "namespace": {
+                    "type": "string"
+                },
+                "container_port": {
+                    "type": "integer"
+                },
+                "docker_url": {
+                    "type": "string"
```

Comment:
now that we've picked a direction: i'd been talking to @EvanKrall about this and he had a pretty nifty idea for getting multi-image support for a given service: we can use the DeploymentVersion code intended for no-commit-redeploys here since the various paasta deployment commands all take `--image-version $VERSION` as an arg - and we could have each toolkit container be in it's own deploy group and have a custom jenkins pipeline to build/tag/push the various images

that said: it's probably not too big a shift to go from docker_url to the image version approach, so i'm personally fine with trying the image version bits later if you'd like to start with this approach

Reply:
(would also be nice since it would stop folks from being tempted to expand where else docker_url could be used :p)